cmd/minikube/cmd/config/kubernetes_version.go

```diff
@@ -34,3 +38,17 @@ func supportedKubernetesVersions() (releases []string) {
 	}
 	return releases
 }
+
+// IsInGithubKubernetesVersions checks whether ver is in the Github list of k8s versions
+func IsInGithubKubernetesVersions(ver string) (bool, error) {
```

Comment:
```suggestion
// IsInGitHubKubernetesVersions checks whether ver is in the GitHub list of K8s versions
func IsInGitHubKubernetesVersions(ver string) (bool, error) {
```

Reply:
Done. I noticed though that there are some constants in pkg/minikube/notify/constants.go with Github instead of GitHub (like GithubMinikubeReleasesURL) are these just typos?

Reply:
Yeah, they should be `GitHub`